src/Banner.js

```diff
@@ -99,8 +98,7 @@ Banner.propTypes = {
 Banner.defaultProps = {
   bg: 'green',
   textAlign: 'left',
-  showIcon: true,
-  theme: theme
+  showIcon: true
```

Comment:
@jxnblk  Removed theme as default prop for banner, as it is a composite component. Thanks!